guides/sessions/session-management.mdx

```diff
@@ -122,17 +75,89 @@ Sessions can also be (implicitly) terminated by exceeding the number of allowed
 Hanko allows 5 concurrent sessions. If a user establishes another session, then the oldest session becomes
 invalid.
 
-<Note>
-    Stateful (server-side) sessions must be enabled (see:
-    [Stateless vs. stateful sessions](#stateless-vs-stateful-sessions)) to configure concurrent session limits.
-</Note>
-
 To configure the allowed amount of concurrent sessions:
 
 1. Log in to [Hanko Cloud](https://cloud.hanko.io) and select your project.
 2. Navigate to `Settings > Session`.
 3. Under `Session limit`, enter the desired amount and click `Save`.
 
+### Limiting cookie retention period
+
+You can configure the retention period of session cookies. Possible configurations are:
+
+- **Persistent** (default): This type sets the `Max-Age` attribute of the cookie to the specified
+[session duration](#limiting-session-duration) (in seconds). The cookie will be automatically deleted from the browser
+once the specified expiry time has been reached.
+- **Session**: This type sets the `Max-Age` attribute of the cookie to 0, resulting in the cookie being deleted from the
+browser when the browser tab or window is closed.
+- **Prompt**: If you use [Hanko Elements](/guides/hanko-elements/introduction) (+1.3) then selecting this type results
+in Hanko Elements displaying a `Stay signed in` checkbox on the login form which allows users to determine the cookie
+retention period: if the checkbox is checked, the cookie will be a `Persistent` cookie, if it is unchecked the
+cookie will be a `Session` cookie.
+
+To configure the cookie retention period:
+
+1. Log in to [Hanko Cloud](https://cloud.hanko.io) and select your project.
+2. Navigate to `Settings > Session`.
+3. Under `Stay signed in (cookie type)`, select the desired type and click `Save`.
+
+## Session self-service
+
+There are a number of configurations options for your tenant that provide control over how users manage their session
+through the profile.
+
+### Allowing session revocation
+
+You can control whether the `Profile` flow permits revocation of sessions (via dedicated `session_delete` action).
+If you use [Hanko Elements](/guides/hanko-elements/introduction) (+1.3) then the
+[Profile component](guides/hanko-elements/profile-component) provides interface elements that  allow users to revoke
+active sessions (except the than the one currently in use).
```

Comment:
Maybe we should clarify that the current session still can be revoked by the logout endpoint, mentioned in the "Explicit user logout" section.